packages/frontend/i18n/src/i18n-completenesses.json

```diff
@@ -1,26 +1,26 @@
 {
-  "ar": 100,
+  "ar": 99,
```

Comment:
The changes in this file appear to be irrelevant.

Reply:
Without this CI will fail and get stuck.